core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -485,54 +497,62 @@ public String toString() {
               + "the journal, logs, and under file storage data (if using local filesystem) "
               + "are written here.")
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
+          .setScope(Scope.ALL)
           .build();
   public static final PropertyKey ZOOKEEPER_ADDRESS =
       new Builder(Name.ZOOKEEPER_ADDRESS)
           .setDescription("Address of ZooKeeper.")
           .setConsistencyCheckLevel(ConsistencyCheckLevel.ENFORCE)
+          .setScope(Scope.ALL)
           .build();
   public static final PropertyKey ZOOKEEPER_CONNECTION_TIMEOUT =
       new Builder(Name.ZOOKEEPER_CONNECTION_TIMEOUT)
           .setDefaultValue("15s") // matches Zookeeper's default
           .setDescription("Connection timeout to use when connecting to Zookeeper")
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
+          .setScope(Scope.MASTER)
```

Comment:
is this `Scope.ALL`? don't we use it on client side?

Reply:
It was used by primary selector to connect to Zookeeper, so it only used by Alluxio masters or job masters.
 I will update the description of this property